modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -1443,8 +1438,6 @@ public void processAttribute(String prefix, String localName, String value)
 
         @Override
         public void processEndElement() throws IOException {
-            super.processEndElement();
```

Comment:
I don't see a warning for it, What setting is warns about calling an empty method?

Reply:
The `processEndElement` in the base class was an empty method that throws `IOException`.  By making it `abstract` the warning is avoided, but this means the call to `super` here is no longer relevant (it wasn't relevant anyway as the base method was empty).

Reply:
I think that we should refrain from making "structural" changes in this pass. Let's keep the warning on and deal with these later.

Reply:
I see that this change has been reverted. That's good.

FWIW, I disagree with the original premise of removing it. The subclass shouldn't have to know or care that the superclass method is empty or not.

I looked through the current version, and no longer have any concerns about this PR. I'll let others formally review.

Reply:
Although there are many ways to implement this kind of inheritance, I do think that what I did here was also a valid solution.  The parent class was already `abstract` -- all I did was to make the `processEndElement` abstract instead of empty, forcing the subtypes to think about it and implement it.  Calling `super` is then no longer allowed.  I did not remove `super` because I knew it was empty, I removed it because it was now `abstract`.

Reply:
> The parent class was already abstract -- all I did was to make the processEndElement abstract instead of empty, forcing the subtypes to think about it and implement it ...
>  I did not remove super because I knew it was empty, I removed it because it was now abstract.

OK, that's a fair point then. And given that, I wouldn't necessarily be opposed to doing something like this as part of a more targeted cleanup rather than a general "remove warnings" sort of cleanup.